client/src/components/App.jsx

```diff
@@ -3,25 +3,53 @@ import React from 'react';
 import Header from './Header';
 import Overview from './Overview';
 import Review from './Review';
+import queryString from 'query-string';
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
```

Comment:
Agreed that this is the top component for now and no props is passed in. However I think the proxy server may pass in data like the id of the home to be rendered, so may be safer to just leave in props.